R/dvhydrograph-render.R

```diff
@@ -8,10 +8,11 @@ createDvhydrographPlot <- function(data) {
   options(scipen=8)
   
   dvData <- parseDVData(data)
```

Comment:
I'm going to hold off on this for the day but would like to discuss with @zmoore-usgs and @lindsaycarr to decide how far we want to go with it.